dcos/dcos_integration_test.go

```diff
@@ -82,3 +83,7 @@ func (d *dcosSecretTest) TestGetSecret(t *testing.T) error {
 
 	return nil
 }
+
+func (d *dcosSecretTest) TestDeleteSecret(t *testing.T) error {
+	return nil
```

Comment:
you could just add the test looking at GetSecret and PutSecret
we can verify it later

Reply:
done